packages/governance/src/types.js

```diff
@@ -404,14 +405,18 @@
  */
 
 /**
- * @callback GetParams - getParams() retrieves a Record containing
- *   keyword pairs with descriptions of parameters under governance.
- * @returns {Record<Keyword,ParamShortDescription>}
+ * @callback GetGovernedVaultParams
```

Comment:
@turadg  governance shouldn't know about vaults.

This looks like it's actually about Vaults, so I'd like to move it. Is there a reason it needs to be here?

Reply:
This was a best effort to get the types consistent. Feel free to improve!

Reply:
I sorta moved it back in #4437